test/functional/android-helper-e2e-specs.js

```diff
@@ -11,26 +7,18 @@ import ADB from 'appium-adb';
 let apiDemos = sampleApps('ApiDemos-debug');
 let appPackage = 'io.appium.android.apis';
 
-/*const should = */chai.should();
+chai.should();
 chai.use(chaiAsPromised);
 
 describe('android-helpers e2e', () => {
-
   describe('installApkRemotely', () => {
-
     it.skip('installs an apk by pushing it to the device then installing it from within', async function () {
```

Comment:
if we want to keep this skip in committed code, we might want an explanation as to why